paper-slider.html

```diff
@@ -483,6 +488,10 @@
         'up pageup end': '_incrementKey'
       },
 
+      attached: function() {
+        this._isRTL = window.getComputedStyle(this)['direction'] === 'rtl';
```

Comment:
Unfortunately without better platform support this is the limitation we have to endure. It seems to me the biggest issue is currently there is no way to force the direction of the slider to be LTR when the page's <html> has a `dir="rtl"`.  I'd suggest we just make it to support if someone wants to override the `dir` it has to be set on paper-slider `<paper-slider dir="ltr">`.